orbs/docker/orb.yml

```
@@ -82,6 +82,10 @@ commands:
         description: "The tag of the image."
         type: string
         default: "git-$CIRCLE_SHA1"
+      optional:
+        description: "Even if trivy command fails, the step won't fail if optional is true"
+        type: boolean
+        default: true
```

```suggestion
        default: false
```

I'd keep blocking by default. If it's non-blocking by default people are unlikely to ever notice any issues.